webSocket/transceiver/transceiver.go

```diff
@@ -136,7 +169,7 @@ func (wt *wsTransceiver) sendAckIfNeeded(connection webSocket.WSConClient, paylo
 	for {
 		timer.Reset(wt.retryDuration)
 
-		err := connection.WriteMessage(websocket.BinaryMessage, counterBytes)
+		err := connection.WriteMessage(websocket.TextMessage, counterBytes)
```

Comment:
TextMessage implies a UTF-8 encoding and it's slower than BinaryMessage. I'd suggest keeping the binary message

Reply:
refactored this part.